### PR TITLE
update approuter version

### PR DESCRIPTION
**Pull request by sidakphull**
`@sap/appruter:12.0.1` has some changes which make deploying `approuter` on Kyma very easy. 

It'd be great if this change is also merged with the `streamlined-mtx` branch.

### CODE REVIEW

**Comment by beckermarc** [comment]
Hi @sidakphull, thanks for opening this PR. Can you please provide a pointer to the release notes of approuter version 12. I'd be interested in reading about the changes :)

File: app/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "approuter",
 	"dependencies": {
-		"@sap/approuter": "^11"
+		"@sap/approuter": "^12"
```
Comment (beckermarc) [inline app/package.json:4]:
```suggestion
		"@sap/approuter": "^13"
```

Comment (sidakphull) [reply app/package.json:4]: yes this works as well.

**Comment by sidakphull** [comment]
> Hi @sidakphull, thanks for opening this PR. Can you please provide a pointer to the release notes of approuter version 12. I'd be interested in reading about the changes :)

Hi Marc, I sent you a message on Teams with the internal links. Let me know if something more is required.

**Review by beckermarc — APPROVED** [review]
Newer version of approuter include an updated xsenv, which better supports service bindings on Kyma / K8s.

**Pull request merged**